posts/2022-05-12-new-logging-functionality.adoc

```diff
@@ -0,0 +1,160 @@
+----
+
+This new functionality is enabled by configuration through either the a bootstrap property, environment variable and/or through the `server.xml`. Note that if there is configuration present in all strategies mentioned then the configuration precedence is the `server.xml` > environment variable > bootstrap property.
+
+*Configuration:*
+
+`Bootstrap.properties` > `com.ibm.ws.logging.stackJoin=true/false`
+`Server.env` > `WLP_LOGGING_STACK_JOIN=true/false`
+`Server.xml` > `<logging stackJoin="true/false" />`
+
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[#run]
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE XX, Java SE 17, Java SE 11, and Java SE 8.
```

Comment:
```suggestion
To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 18, Java SE 17, Java SE 11, and Java SE 8.
```